### PR TITLE
add atReferenceCounter, rlPlatform

### PR DESCRIPTION
**Pull request by PliskinDev**

### CODE REVIEW

File: network/rlPlatform.hpp [hunk 1]
```diff
@@ -0,0 +1,48 @@
...
+                info->Clear();
+            }
+            return info;
+        }
+        */
+    private:
+        const char* m_displayName = "DefaultName";
+    };
+
+    class rlPlatformParty
```
[the remark this answers is not in the document]
Comment (gir489returns) [reply network/rlPlatform.hpp:33]: This entire class is clearly stolen from the leaked source code. Knowing what pre-compiler arguments like noexcept and if the arguments are const are not is not something you can extrapolate from IDA. This was 100% not reverse engineered. Take the leaked source and jam it up your ass.

**Comment by gir489returns** [comment]
We warned you before not add leaked shit to the repo and now you're doing it again. https://github.com/YimMenu/RDR-Classes/pull/6

If you do it again, we will ban you from the repository. Consider this your final warning.

**Comment by Rxann** [comment]
Thank you @gir489returns and @maybegreat48 for catching this. I appreciate it a lot. Refer to their comments.

File: network/rlPlatform.hpp (hunk 1 above)
[the remark this answers is not in the document]
Comment (PliskinDev) [reply network/rlPlatform.hpp:33]: > This entire class is clearly stolen from the leaked source code. Knowing what pre-compiler arguments like noexcept and if the arguments are const are not is not something you can extrapolate from IDA. This was 100% not reverse engineered. Take the leaked source and jam it up your ass.

watch your mouth please
leaked and stolen? what are you talking about? classes. I already dumped them before r*  made a strip for them, and I also don't want to go into Ida's details.if you don't like the work, say I don't like it or I don't want to see you again and that's it
I can work alone and this project becomes as dead as before , It was a pleasure working with you all bye

**Pull request closed without merging**